src/ConnectionAutomaton.cpp

```diff
@@ -320,13 +320,13 @@ void ConnectionAutomaton::onConnectionFrame(
   switch (type) {
     case FrameType::KEEPALIVE: {
       Frame_KEEPALIVE frame;
-      if (!deserializeFrameOrError(frame, std::move(payload))) {
+      if (!deserializeFrameOrError(isResumable_, frame, std::move(payload))) {
```

Comment:
Introspecting the frame doesn't even work right? We can't actually know if we're looking at the data size or if we're looking at the ResumePosition?

Given that all gateways should now run with resumability isn't this change fine to make?